metal3-templater.go

```diff
@@ -23,13 +23,19 @@ type templater struct {
 	BootstrapInspectorURL string
 	ClusterIronicURL      string
 	ClusterInspectorURL   string
+	IronicUser            string
+	IronicPassword        string
+	InspectorUser         string
+	InspectorPassword     string
 }
 
 func main() {
 	var templateData templater
-
-	if len(os.Args) < 7 {
-		fmt.Printf("usage: <prog> TEMPLATE_FILE INTERFACE NETWORK BOOTSTRAP_IP CLUSTER_IP IMAGE_URL\n")
+	numArgs := len(os.Args)
+	if numArgs == 7 && numArgs == 11  {
```

Comment:
It seems unlikely that this expression will ever be true.

Reply:
sorry it should be an OR... I thought I changed..